packages/bpk-stylesheets/README.md

```diff
@@ -8,14 +8,16 @@ Check the main [Readme](https://github.com/skyscanner/backpack#usage) for a comp
 
 ## Usage
 
-```scss
-@import '~@skyscanner/backpack-web/bpk-stylesheets';
+Within your Javascript (usually the entrypoint or top-most JS file)
+
+```js
+import 'bpk-stylesheets';
```

Comment:
don't we need the `@skyscanner/backpack-web` prefix?

Reply:
🤦yes you are right! Good catch